api/apps/articles/model/retrieve.coffee

```diff
@@ -47,6 +48,10 @@ schema = require './schema'
 
     # Allow regex searching through the q param
     query.thumbnail_title = { $regex: new RegExp(input.q, 'i') } if input.q
+
+    # Look for articles with scheduled dates before the given date
+    query.scheduled_publish_at = { $lt: moment(input.scheduled_publish_at).toDate() } if input.scheduled_publish_at
```

Comment:
Adding this to `where` exposes it in the API—which is fine, but I might call this `scheduled_publish_before` for clarity of end-user sake. You can also just use `db.find({ $lt ... })` instead of `@where` inside of `publishScheduledArticles` instead.